.github/workflows/build-client.yaml

```diff
@@ -59,6 +61,7 @@ jobs:
         id: docker_build
         uses: docker/build-push-action@v3
         with:
+          load: false
```

Comment:
Yes (we have to so that cass-operator can ship with correct versions), but that would come in a separate PR. We might also want to publish just a static binary for certain platforms if we add more CLI user features.